model/src/main/proto/javaclasses/mealorder/values.proto

```diff
@@ -32,29 +32,44 @@ option java_generate_equals_and_hash = true;
 
 import "google/protobuf/timestamp.proto";
 
-//TODO: Change to another class!
-message EmailAdress {
+//TODO: Use Spine EmailAddress!
+// The model that represents the email.
+//
+message EmailAddress {
 
     string value = 1;
 }
-
+// The model that represents date range.
+//
 message MenuDateRange {
 
-    google.protobuf.Timestamp start_range = 1;
+    // Range start date.
+    google.protobuf.Timestamp when_range_started = 1;
 
-    google.protobuf.Timestamp end_range = 2;
+    // Range end date.
+    google.protobuf.Timestamp when_range_ended = 2;
 }
 
+// The model that represents phone number.
+//
 message PhoneNumber {
 
-    repeated string values = 1;
+    string values = 1;
 }
 
+// The model that represents vendor description.
+//
+// Vendor description it is email, pnone numbers and daily deadline.
+// After reaching deadline, the order will not be accepted.
+//
 message VendorDescription {
 
-    EmailAdress email = 1;
+    // Email adress  to send an purchase order.
+    EmailAddress email = 1;
 
-    PhoneNumber phone_number = 2;
+    // Phone numbers of the vendor.
+    repeated PhoneNumber phone_numbers = 2;
 
+    // A daily deadline.
     google.protobuf.Timestamp po_daily_deadline = 3;
```

Comment:
It's not `Timestamp`. It should be `LocalTime`.

Reply:
`Timestamp` is a point on timeline. RTFM.

Reply:
Do you mean something like this?

```
message LocalTime{
  int32 hours = 1;

  int32 minutes = 2;
}
```